### PR TITLE
fix(release-notes): added release notes for 2022.01

### PR DESCRIPTION
**Pull request by mcoker**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,46 @@ id: Release notes
...
+- **Text input group:** Added autocomplete ghosting ([#4616](https://github.com/patternfly/patternfly/pull/4616))
+- **Truncate:**
+  - Resolved spacing issues ([#4599](https://github.com/patternfly/patternfly/pull/4599))
+  - Fixed font size var value ([#4560](https://github.com/patternfly/patternfly/pull/4560))
+
+### Other
+- **Docs:**
+  - Promoted beta components ([#4613](https://github.com/patternfly/patternfly/pull/4613))
+  - Removed outdated modifiers page ([#4619](https://github.com/patternfly/patternfly/pull/4619))
+- **Global:** Udpated/removed unused/undefined vars ([#4620](https://github.com/patternfly/patternfly/pull/4620))
```
Comment (nicolethoen) [inline RELEASE-NOTES.md:40]: typo in `updated`

**Review by nicolethoen — COMMENTED** [review]
small type found - but dude! The diffs for the PNGs are so awesome! Have I just not noticed that before?

File: RELEASE-NOTES.md (hunk 1 above)
Comment (mcoker) [reply RELEASE-NOTES.md]: udpated!!

**Comment by mcoker** [comment]
> The diffs for the PNGs are so awesome!

That's rad! I'm assuming it's new, I believe it was just a typical binary file diff before.

**Pull request merged**